test/templates/counter/index.njk

```diff
@@ -1,70 +1,41 @@
 {% extends "layout.njk" %}
-{% from "./component.njk" import counter %}
 {% from "./../format.njk" import format %}
 
 {% block content %}
-<div>
-
-    <h2>default</h2>
-    <p>Show nothing when 0, max is 999</p>
-    <br>
+    <style>
+        .pg-icon-placeholder::before {
+            border: 1px solid;
+            padding: 5px;
+            content: "i";
+        }
+    </style>
+
+    <h2>default counters</h2>
+    <br/>
     {% set single_counter %}
-{{ counter(count=0, modifier="") }}
-{{ counter(count=5, modifier="") }}
-{{ counter(count=25, modifier="") }}
-{{ counter(count=101, modifier="") }}
-{{ counter(count=4001, modifier="") }}
+    <span class="fd-counter" aria-label="Unread count">5</span>
+    <span class="fd-counter" aria-label="Unread count">25</span>
+    <span class="fd-counter" aria-label="Unread count">101</span>
+    <span class="fd-counter" aria-label="Unread count">4001</span>
+    <span class="fd-counter" aria-label="Unread count">9999+</span>
```

Comment:
I think these should all be changed back to use the counter imported from the `component.njk` file.  I know they are very simple, but this keeps it consistent with all the other playground page constructs.